openstudiocore/src/utilities/units/Scale.cpp

```diff
@@ -114,6 +114,20 @@ namespace openstudio{
     static Scale M = {"M", "mega",6,1.0E6};
     return M;
   }
+  
+  /** Static constant defining SI prefix for 10^5. */
+  UTILITIES_API const Scale& hectokilo()
+  {
+    static Scale hk = {"_hk", "hectokilo",5,1.0E5};
+    return hk;
+  }
+
+  /** Static constant defining SI prefix for 10^4. */
```

Comment:
Well the main thing is we need a named scale for 1E4, I don't really care what it is named.  The actual short name used is "_ma" which should be enough to prevent it from being found in any unit strings, k is just the local variable.